### PR TITLE
Remove ssh-keyscan from gen-edpm-compute-node script

### PR DESCRIPTION
**Pull request by arxcruz**
The ssh-keyscan will fail if there is no info on known_hosts for the ip address 192.168.122.1 and so the virt-customize will command fail to run, specially when you're trying to deploy locally in your machine instead of using nested vms.

### CODE REVIEW

**Comment by cjeanner** [comment]
/approve

Note: this ssh-keyscan was a forgotten bit when I cleaned the ssshuttle related tasks/content. We don't need to inject the hypervisor host key in the VM.

File: devsetup/scripts/gen-edpm-compute-node.sh [hunk 1]
```diff
@@ -181,6 +181,8 @@ sed -i s/dhcp/none/g $NETSCRIPT
 sed -i /PERSISTENT_DHCLIENT/d $NETSCRIPT
 EOF
 
+chmod +x ${OUTPUT_BASEDIR}/${EDPM_COMPUTE_NAME}-firstboot.sh
```
Comment (jpodivin) [inline devsetup/scripts/gen-edpm-compute-node.sh:184]: I'm not 100% sure how is this related to the ssh-keyscan removal. Maybe it would be a good idea to make a note of it in PR or even in the commit msg.

Comment (arxcruz) [reply devsetup/scripts/gen-edpm-compute-node.sh:184]: Updated the commit message.

**Comment by arxcruz** [comment]
recheck

**Comment by arxcruz** [comment]
recheck
